Cartfile

```diff
@@ -1,2 +1,4 @@
-github "mapbox/mapbox-vision-ios" == 0.2.0
-github "mapbox/mapbox-navigation-ios" ~> 0.26.0
+github "mapbox/mapbox-vision-ios" "kobe/minor-bump-vision-sdk-0.3.0"
```

Comment:
@chezzdev when https://github.com/mapbox/mapbox-vision-ios/pull/54 gets approved, we need to change this to `~>0.3.0`
/cc @akitchen @okoriep @atripathi-mb @d-prukop @andrewychen